build.gradle

```diff
@@ -2,7 +2,7 @@
 
 buildscript {
 
-    ext.kotlin_version = '1.3.61'
+    ext.kotlin_version = '1.3.72'
```

Comment:
Need to test this with #98

Reply:
Might be good to know what Kotlin version AOSP10 uses so we can stay in sync with that.

Reply:
1.3.21 in the latest Android 10 release (august)
1.3.72 in master, let's see what R / 11 has.

Source: https://android.googlesource.com/platform/external/kotlinc/+/refs/tags/android-10.0.0_r41/build.txt

Reply:
Hmm... so maybe it is better to downgrade to that? I *think* that all 1.3.x should be compatible, but who knows...

Reply:
Not sure. So far it hasn't been an issue, but yeah might be better to stay in sync.

But then again 11 isn't too far out, so let's decide then?

Reply:
Ok so I reverted this change and left the Kotlin version as it was.